canonical/microk8s.io
# [WD-7202] remove support phone numbers

## Done

Removed phone numbers from /contact-us page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Navigate to /contact-us and check if the `Call us` section has been removed


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6936

## Screenshots

[if relevant, include a screenshot]
